src/main/java/com/user_service/dtos/response/SignInResponseDto.java

```diff
@@ -0,0 +1,8 @@
+package com.user_service.dtos.response;
+
+public record SignInResponseDto(String accessToken,
+                                String tokenType,
+                                String refreshToken,
+                                String idToken,
+                                Integer expiresIn) {
```

Comment:
What's the unit? It's good to include it in the name, e.g. `expiresInMin` or `expiresInMs`